### ISSUE TITLE
Release new version to npm

### ISSUE THREAD

**Issue by swrobel** [renchap/webpacker-react#19]
The currently-released version, `0.1.0` contains a bug that causes `configureHotModuleReplacement()` to break. It has since been fixed in a [commit](https://github.com/renchap/webpacker-react/commit/9e8e632489a757794ba813f91f0d19753346dd3a#diff-2b0da08e38daf70aa89f0c2153adf93cL4) after that package was released. A new release should be in order. Here is the error that the current module produces:

```
TypeError: Cannot read property 'rules' of undefined
    at configureHotModuleReplacement (node_modules/webpacker-react/configure-hot-module-replacement.js:22:38)
```

**Comment by renchap** [comment]
I am using the released 0.1.0 version and not seeing any issue. The commit you pointed is only syntax changes.

**Comment by renchap** [comment]
This commit should fix it. Webpacker changed the default Webpack config, I updated our docs.
